.github/CONTRIBUTING.md

```diff
@@ -44,8 +44,8 @@ your contributions.
 
 * All code must also pass [hlint](http://community.haskell.org/~ndm/hlint/) and [stylish-haskell](https://github.com/jaspervdj/stylish-haskell)
   with no warnings. This helps enforce a uniform style for all committers. Continuous integration will check this as well on every
-  pull request. There's a useful Makefile that helps with checking this locally. You can run `make commit-check` to do this manually but
-  we recommend adding it to `.git/hooks/pre-commit` to automatically check this before doing a commit.
+  pull request. There are useful tools in the nix-shell that help with checking this locally. You can run `postgrest-check` to do this manually but
```

Comment:
`postgrest-check` is not an exact replacement of `make commit-check`. The latter only did style and lint, while the former also does spec tests in addition.

Guys, do you think it is ok like that or do we need another `postgrest-commit-check` here? @steve-chavez @monacoremo

Reply:
Would be an improvement IMO!

Reply:
Which one exactly would be the improvement? The `postgrest-check` as is or the `postgrest-commit-check`? :D

Reply:
Sorry, that was unclear - I meant leaving it at postgrest-check and having the extended checks by default. If anyone wants to check less, they can always use the specific commands

Reply:
@steve-chavez If you think otherwise, we can still add it later. Everything else is addressed now, so I'm merging this one as well.